webpack.config.js

```diff
@@ -47,7 +46,7 @@ module.exports = {
     // Only load the locales we need to support in moment.js
     new webpack.ContextReplacementPlugin(
       /moment[\/\\]locale$/,
-      new RegExp(SETTINGS.SUPPORTED_LANGUAGES.join('|')),
+      new RegExp(['de', 'en', 'fr'].join('|')),
```

Comment:
Duplication for the sake of simplicity.

Reply:
But you are "hardcoding" available languages ?

Reply:
Yes. I can't take them from the settings anymore because settings are a TS file and webpack conf can't import TS.